projects/Lehigh_Lang.yaml

```diff
@@ -0,0 +1,11 @@
+Department: Department of Biological Sciences
+Description: Our research focuses on understanding the dynamics of mitonuclear co-evolution
+  in populations of microorganisms. We use experimental evolution, next-generation
+  sequencing, and systems biology approaches to study how populations adapt to different
+  pairs of nuclear and mitochondrial genomes to uncover the mechanisms underlying
+  these processes.
```

Comment:
Description: >-
  Our research focuses on understanding the dynamics of mitonuclear
  co-evolution in populations of microorganisms. We use experimental
  evolution, next-generation sequencing, and systems biology
  approaches to study how populations adapt to different pairs of
  nuclear and mitochondrial genomes to uncover the mechanisms
  underlying these processes.